logs/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Why do you specify the exclusion of all files (except ```.gitignore```) in here and in the ```.gitignore``` one level above?